### PR TITLE
Telesci Fixes

### PR DESCRIPTION
**Pull request by fludd12**
## About The Pull Request

Fixes a bunch of telesci issues, including teleportation inhibitors unable to be deconstructed!
Also adds in some truly beautiful icons by @Michiyamenotehifunana and @Sigmasquad!
Also-also reworks the unused pod bay below R&D into a usable but sparsely-outfitted auxiliary research section.

## Why It's Good For The Game

i cant believe it took me so long to get this out im so sorry

## Changelog
```changelog
add: New telescience icons! Thank you Tori and Sigma!
add: The pod bay below R&D has been reworked into an Auxilliary Research section, open to the public.
soundadd: There's a number of new sounds associated with telescience, to add to the audio/visual cues!
tweak: Telescience pads can no longer be controlled by more than once console at once.
fix: Telescience inhibitors can now be deconstructed.
```

### CODE REVIEW

**Comment by fludd12** [comment]
Sorry for the excessive number of commits!

**Comment by Shadow-Quill** [comment]
> Sorry for the excessive number of commits!

I have seen *much* worse.

File: zzzz_modular_occulus/code/modules/telescience/telepad_computer.dm [hunk 1]
```diff
@@ -110,29 +113,53 @@
...
 			relay.chanceExplode()
+		var/inhibitorExploded = FALSE
+		for(var/obj/machinery/telesci_inhibitor/blocker in portalarea.tele_inhibitors)
+			if(!istype(blocker))
+				continue
+			inhibitorExploded = TRUE
+			playsound(telegraph,'zzzz_modular_occulus/sound/effects/telesci_inhibitor_alarm.ogg', 80, FALSE, 7, extrarange = 10, is_global = FALSE, ignore_walls = TRUE)
+			blocker.visible_message(SPAN_DANGER("\The [src] sparks violently and begins to shake!"))
+			do_sparks(6, FALSE, get_turf(blocker))
+			addtimer(CALLBACK(blocker, /obj/machinery/telesci_inhibitor/proc/explode), 1 SECONDS)
```
Comment (Shadow-Quill) [inline zzzz_modular_occulus/code/modules/telescience/telepad_computer.dm:137]:
```suggestion
			addtimer(CALLBACK(blocker, /obj/machinery/telesci_inhibitor/proc/explode), 1 SECOND)
```
\>:C

**Review by Shadow-Quill — APPROVED** [review]
Overall, code looks fine but I have a nitpick to make.

File: zzzz_modular_occulus/code/modules/telescience/telepad_computer.dm (hunk 1 above)
Comment (fludd12) [reply zzzz_modular_occulus/code/modules/telescience/telepad_computer.dm:137]: I did not know someone made a `SECOND` define specifically to allow for grammatically correct code, that's really funny.

**Comment by fludd12** [comment]
Map render check failed but it seems to be not on my end, so that's why it's failing checks.

**Review by Shadow-Quill — APPROVED** [review]

File: zzzz_modular_occulus/code/modules/telescience/telepad_computer.dm [hunk 2]
```diff
@@ -143,13 +144,16 @@
 
 
 	qdel(telegraph)
```
Comment (Shadow-Quill) [inline zzzz_modular_occulus/code/modules/telescience/telepad_computer.dm:146]: You can replace the following four lines with
`QDEL_NULL(thing)`
C:

File: zzzz_modular_occulus/code/modules/telescience/telepad_computer.dm [hunk 3]
```diff
@@ -298,6 +302,8 @@
 	ticking = FALSE
 	menuOption = BS_MENU_SELECT
 	qdel(telegraph)
```
Comment (Shadow-Quill) [inline zzzz_modular_occulus/code/modules/telescience/telepad_computer.dm:304]: Ditto

**Review by Shadow-Quill — CHANGES_REQUESTED** [review]

File: zzzz_modular_occulus/code/modules/telescience/telepad_computer.dm [hunk 4]
```diff
@@ -110,29 +114,56 @@
...
+		if(inhibitorExploded)
+			var/obj/item/device/radio/headset/a = new /obj/item/device/radio/headset(null)
+			a.autosay("ALERT: Extreme bluespace disruption detected in [loc]. Equipment failure imm-m-...", "Bluespace Inhibition Node")
+			qdel(a)
+
+
 	qdel(telegraph)
+	qdel(construct)
 	telegraph = null
+	construct = null
```
Comment (Shadow-Quill) [inline zzzz_modular_occulus/code/modules/telescience/telepad_computer.dm:149]:
```suggestion
	QDEL_NULL(telegraph)
	QDEL_NULL(construct)
```

File: zzzz_modular_occulus/code/modules/telescience/telepad_computer.dm [hunk 5]
```diff
@@ -271,7 +302,13 @@
 	ticking = FALSE
 	menuOption = BS_MENU_SELECT
 	qdel(telegraph)
+	qdel(construct)
+	construct = null
 	telegraph = null
```
Comment (Shadow-Quill) [inline zzzz_modular_occulus/code/modules/telescience/telepad_computer.dm:307]:
```suggestion
	QDEL_NULL(telegraph)
	QDEL_NULL(construct)
```

**Review by Shadow-Quill — COMMENTED** [review]
Doop

**Review by Shadow-Quill — APPROVED** [review]

**Pull request merged**